### PR TITLE
january udates

### PR DESCRIPTION
**Pull request by xrotwang**
Fixes in Sinitic, Swedic, Kikongo and a few more items.

### CODE REVIEW

File: languoids/tree/sino1245/sini1245/sout2740/ping1244/ping1244.ini [hunk 1]
```diff
@@ -1,31 +0,0 @@
...
-hid = NOCODE_Pinghua
-level = language
-latitude = 27.6725
-longitude = 112.101667
-macroareas = 
-	Eurasia
-countries = 
-	China (CN)
-
-[classification]
```
Comment (xrotwang) [inline languoids/tree/sino1245/sini1245/sout2740/ping1244/ping1244.ini]: Do we need to move this information somewhere else? I.e. is there a new languoid for which this info is appropriate?

**Comment by d97hah** [comment]
Pinghua ping1244 should no longer be the level language, but the level
family, while S Pinghua and N Pinghua are now the language-level languoids.
The other information is correct but need not be carried over (because it's
already taken care of). The new state of affairs is encoded in lff/dff but
perhaps not carried over into the level field in the .ini:s. There should
be a few more cases like this in the updates the last two months.

2017-01-23 13:26 GMT+01:00 Robert Forkel <notifications@github.com>:

**Comment by xrotwang** [comment]
Ok. So I guess we can merge and start over :)

**Comment by d97hah** [comment]
Yes. Should I do anything about the level issue?

2017-01-23 13:41 GMT+01:00 Robert Forkel <notifications@github.com>:

**Comment by xrotwang** [comment]
Levels should be set correctly by lff2tree - even for existing languoids. The copying isn't really copying, but rather 
- read ini file into `Languoid` object
- possibly correct level
- write `Languoid` object to new ini file

**Pull request merged**